feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -49,6 +52,44 @@ class IdentityManager(
         }
     }
 
+    fun logIn(
+        newAppUserID: String,
+        onSuccess: (PurchaserInfo, Boolean) -> Unit,
+        onError: (PurchasesError) -> Unit
+    ) {
+        if (newAppUserID.isBlank()) {
+            onError(PurchasesError(
+                PurchasesErrorCode.InvalidAppUserIdError,
+                "appUserID can't be null, empty or blank"
+            ).also { errorLog(it) })
+            return
+        }
+
+        log(LogIntent.USER, IdentityStrings.LOGGING_IN.format(currentAppUserID, newAppUserID))
+        val oldAppUserID = currentAppUserID
+        backend.logIn(
+            oldAppUserID,
+            newAppUserID,
+            { purchaserInfo, created ->
+                synchronized(this@IdentityManager) {
+                    log(
+                        LogIntent.USER,
+                        IdentityStrings.LOG_IN_SUCCESSFUL.format(newAppUserID, created)
+                    )
+                    deviceCache.clearCachesForAppUserID(oldAppUserID)
+                    subscriberAttributesCache.clearSubscriberAttributesIfSyncedForSubscriber(
+                        oldAppUserID
+                    )
```

Comment:
Being nitpicky here, but the line breaks are not needed
```suggestion
                    subscriberAttributesCache.clearSubscriberAttributesIfSyncedForSubscriber(oldAppUserID)
```

Reply:
the formatter is set to 120 characters, so it auto-adds them. I'll fix for now, but maybe we should have a strict set of rules for this? I thought detekt was set to 120 as well

Reply:
120 is good, that's what I have too. My formatter is not auto-adding the breaks. Maybe there is another config thing going on? Let me know if you want to compare settings to figure out what's going on

Reply:
yeah, would love to have shared settings. we should probably even share them somewhere so they can be reused by other RC members

Reply:
you can export just the code style stuff 
<img width="572" alt="image" src="https://user-images.githubusercontent.com/3922667/108532954-2e854680-72b7-11eb-8f1b-262f64209f4e.png">